configs/cs-en-spring-2024.yml

```diff
@@ -0,0 +1,236 @@
+  #  - mtdata_ParaCrawl-paracrawl-8-eng-ces - duplicate with opus
+  #  - mtdata_ParaCrawl-paracrawl-9-eng-ces - duplicate with opus
+  #  - mtdata_Statmt-europarl-9-ces-eng - duplicate with opus
+  #  - mtdata_Statmt-europarl-7-ces-eng - duplicate with opus
+  #  - mtdata_Statmt-news_commentary-14-ces-eng - duplicate with opus
+  #  - mtdata_Statmt-news_commentary-15-ces-eng - duplicate with opus
+  #  - mtdata_Statmt-news_commentary-16-ces-eng - duplicate with opus
+  #  - mtdata_Statmt-europarl-10-ces-eng - duplicate with opus
+  #  - mtdata_Statmt-ccaligned-1-ces_CZ-eng - duplicate with opus
+  #  - mtdata_Tilde-ecb-2017-ces-eng - duplicate with opus
+  train:
+  - opus_NLLB/v1  #                                       56,307,029 sentences
+  - opus_ParaCrawl/v9 #                                  50,633,505 sentences
+  - opus_OpenSubtitles/v2018 #                           42,346,436 sentences
+  - opus_StanfordNLP-NMT/v1.0 #                          15,793,121 sentences
+  - opus_ELRC-EMEA/v1 #                                  12,891,707 sentences
+  - opus_CCAligned/v1 #                                  12,730,121 sentences
+  - opus_DGT/v2019 #                                      5,207,753 sentences
+  - opus_LinguaTools-WikiTitles/v2014 #                   4,813,030 sentences
+  - opus_XLEnt/v1.2 #                                     3,894,132 sentences
+  - opus_JRC-Acquis/v3.0 #                                1,273,411 sentences
+  - opus_ELRC-5067-SciPar/v1 #                            1,064,385 sentences
+  - opus_EMEA/v3 #                                        1,053,385 sentences
+  - opus_ELRC-2713-EMEA/v1 #                                779,083 sentences
+  - opus_ELRC_2682/v1 #                                     779,082 sentences
+  - opus_Europarl/v8 #                                      647,095 sentences
+  - opus_WikiMatrix/v1 #                                    519,195 sentences
+  - opus_EUbookshop/v2 #                                    455,472 sentences
+  - opus_QED/v2.0a #                                        441,508 sentences
+  - opus_ELITR-ECA/v1 #                                     295,788 sentences
+  - opus_Tanzil/v1 #                                        233,399 sentences
+  - opus_News-Commentary/v16 #                              218,509 sentences
+  - opus_TED2020/v1 #                                       170,611 sentences
+  - opus_wikimedia/v20230407 #                              146,717 sentences
+  - opus_KDE4/v2 #                                          134,071 sentences
+  - opus_ELRC-presscorner_covid/v1 #                        129,652 sentences
+  - opus_NeuLab-TedTalks/v1 #                               111,107 sentences
+  - opus_ECB/v1 #                                            63,716 sentences
+  - opus_bible-uedin/v1 #                                    62,151 sentences
+  - opus_WMT-News/v2019 #                                    44,859 sentences
```

Comment:
We should remove https://opus.nlpl.eu/WMT-News This is a dev/test set

Reply:
Actually, for some languages, it's pretty big so maybe it's not the test set but it's not clear from https://www.statmt.org/wmt19/translation-task.html that there was an extra dataset released that can be used for training. \

Reply:
sacrebleu wmt19 for en-lt is 1000 lines long and this one is 5,998 so it's likely something else

Reply:
I wonder if we should have a step the removes any devtest or test from the training data, so we make sure we're not lying to ourselves.